README.md

```diff
@@ -446,6 +446,23 @@ WorkmanagerPlugin.setPluginRegistrantCallback { registry in
 ```
 to [AppDelegate.swift](example/ios/Runner/AppDelegate.swift)
 
+### Request Pin Widget
+Requests pin of HomeWidget to home screen directly from the app.
```

Comment:
Maybe add a quick sentence that that would add the Widget to the Homescreen. (I had to read that Android Docs blurb twice to understand that `pinning == adding`)
```suggestion
Requests to add the Widget to the users HomeScreen by pinning it to the Users HomeScreen.
```

Reply:
I think we should left "pin" word here as well, to match Android's native world, something like 

````
Requests to add (Android's pin widget feature) the Widget to the users HomeScreen by pinning it to the Users HomeScreen. 
```
or something like that

Reply:
Maybe let's go with the wording I proposed in the other file to have `Pin (Add)`.

Good point of keeping the native wording. Thanks @petrnymsa